src/check_jsonschema/catalog.py

```diff
@@ -98,6 +98,16 @@ def _githubusercontent_url(owner: str, repo: str, ref: str, path: str) -> str:
             "types_or": ["json", "yaml"],
         },
     },
+    "codecov": {
+        "url": "https://www.schemastore.org/codecov.json",
+        "hook_config": {
+            "name": "Validate Codecov config",
+            "files": [
+                r"^((\.github|dev)/)?\.?codecov\.ya?ml$",
```

Comment:
Can you give me a pointer to codecov docs for this which indicate that `.github/codecov.yaml` is supported?

The reason I raise this is that I prefer to follow the lead of SchemaStore whenever possible (i.e. as long as we don't have reason to believe that SchemaStore is wrong). It makes for a much simpler policy around the file paths.

SchemaStore [currently lists](https://github.com/SchemaStore/schemastore/blob/98eeb8ccb7cbf6617c848136a0041a197fd72678/src/api/json/catalog.json#L1555) only `codecov.yml` and `.codecov.yml` in the repo root. If that's wrong, I can accept this PR and work on upstreaming the improvement there (or you can give them a PR if you like -- I've always found them very friendly!).

Reply:
https://docs.codecov.com/docs/codecov-yaml#can-i-name-the-file-codecovyml

Please submit a PR to the upstream; I don’t intend to submit one.

Reply:
Awesome, thanks! I'll make sure your work finds its way up into SchemaStore.